test/language/expressions/exponentiation/exponentiation-operator-accessor-invocation.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2015 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Rick Waldron
+email: waldron.rick@gmail.com
+es7id: pending
+description: >
+    Performs exponential calculation on operands. Same algorithm as %MathPow%(x, y)
+---*/
+
+
+var calls = 0;
+var q = {q: 3};
+var o = {
+  get p() {
+    calls++;
+    return q;
+  }
+};
+
+o.p.q **= 2;
+
+assert.sameValue(calls, 1, "Expected o.p.q accessor to be called once");
+assert.sameValue(o.p.q, 9, "Expected (o.p.q **= 2) to be 9");
```

Comment:
this o.p.q test seems more appropriated in a syntax test file, where it gets the value from the lefthandexpression.